src/views/macros/showDefaultAndVariants.njk

```diff
@@ -33,7 +33,7 @@
 
 <h4 class="govuk-u-copy-19">Macro</h4>
 <pre><code>
-{% raw %}{{ loadComponentTemplate(componentName, {% endraw %}
+{% raw %}{{ {% endraw %} govuk{{ componentName | capitaliseComponentName }}(
```

Comment:
We end up with a double space here – there's a space after the `{{` and another one before `govuk`.